src/workerd/server/server.c++

```diff
@@ -1340,25 +1392,25 @@ public:
 
           TimerChannel& timerChannel = service;
 
-          auto loopback = kj::refcounted<Loopback>(*this, Worker::Actor::cloneId(id));
+          auto loopback = kj::refcounted<Loopback>(*this, kj::str(id));
           Worker::Lock lock(*service.worker, asyncLock);
           auto newActor = kj::refcounted<Worker::Actor>(
-              *service.worker, nullptr, kj::mv(id), true, kj::mv(makeActorCache),
+              *service.worker, nullptr, kj::str(id), true, kj::mv(makeActorCache),
```

Comment:
Prior to this change, `id` was a `Worker::Actor::Id`, but this downgraded it to a string. Oops. This is visible in `ctx.id` in JavaScript having the wrong type (a string, rather than a DO ID), but it seems nobody ever noticed. Fixed in #4041.